tests/backend/test_tf_ops.py

```diff
@@ -16,7 +16,7 @@
 import pytest
 
 import numpy as np
-tf = pytest.importorskip("tensorflow", minversion="2.0")
+import tensorflow as tf
```

Comment:
pls revert this merge, it is overwritting a step @josh146 asked for in code review

Reply:
Oh, my bad. Is it OK to just change this line back to `tf = pytest.importorskip("tensorflow", minversion="2.0")`, or are you suggesting somehow reverting the merge? Not sure exactly how that's done.

Reply:
all I'm suggesting is undoing this change. How you achieve it is up to you :laughing: